src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -53,7 +53,7 @@ WARNING:   When setting properties based on their current state (for example:
   <Target Name="GenerateBashExecutionScript"
     Inputs="$(MSBuildProjectFullPath)"
     Outputs="$(OutputPath)\$(AssemblyName).sh"
-    DependsOnTargets="$(BashScriptSnippetGen);GetIlasmRoundTripBashScript">
+    DependsOnTargets="$(BashScriptSnippetGen);GetIlasmRoundTripBashScript;GetSuperPMICollectionBashScript">
```

Comment:
I am wondering how `Get*BashScript` are consumed?

Reply:
This specifies a dependency on a `Get*` target that causes that target to run, which defines a property, e.g., `$(IlasmRoundTripBashScript)`, which gets included in the generated script.